basket.js

```diff
@@ -9,9 +9,11 @@ const Basket = {
 
   async subscribe(email) {
     const params = {
+      form: true,
```

Comment:
I can't tell what this is doing. It appears to be sending `form=true` to basket in the POST, but I don't think that will affect anything.

Reply:
Oh! this is an option to got. I think this should go down in the reqOptions object.

Reply:
Moved.